specs/sb/access-control.md

```diff
@@ -266,12 +266,12 @@ In this example, the GUID of my-org is c54bf317-d791-4d12-89f0-b56d0936cfdc.
 
 To make a private plan available to a specific organization, run:
 
-`cf curl /v2/service_plan_visibilities -X POST -d '{"service_plan_guid":"SERVICE_PLAN_GUID","organization_guid":"ORG_GUID"}'`
+`cf curl /v2/service_plan_visibilities -X POST -d '{"service_plan_guid":"SERVICE_PLAN_GUID","context_guid":"CONTEXT_GUID"}'`
```

Comment:
This is a cloud controller endpoint used for access control, and is unrelated to the service broker.